App.tsx

```diff
@@ -15,7 +15,9 @@ import { ConfigurationProvider } from "./src/ConfigurationContext"
 import { PermissionsProvider } from "./src/PermissionsContext"
 import { initializei18next, loadUserLocale } from "./src/locales/languages"
 
-Bugsnag.start()
+if (env.STAGING === "true" || __DEV__) {
```

Comment:
I believe that this check will also be needed then on the logger file since we could try to call `Bugsnag` on the wrong environment, maybe we could move this to an initializer on that class that sets a logger instance that calls to `Bugsnag` on staging and dev and uses the console otherwise.

Reply:
I think we are also missing the android side to not instantiate the bugsnag native library

Reply:
Android disables Bugsnag on release builds, so only staging and dev should have it enabled

Reply:
awesome, thanks @juancruzgs

Reply:
@aledustet , went for a quick fix, i think we can revisit the initialization pattern in a future commit.